### PR TITLE
important book updates

### PR DESCRIPTION
**Pull request by sachayves**
update introduction, medalla and walletstore guides, and FAQ chapter (copied over the work i did for the medalla launchpad)

change layout (make it clear which sections are addressed to advanced users)
edits, simplifications, and clarifications throughout

### CODE REVIEW

**Comment by mratsim** [comment]
I've proposed in the past to rename the repos: Nimbus -> Nimbus-eth1 and nim-beacon-chain -> Nimbus-eth2 but it was argued that it wouldn't be needed once people have habits.

Another alternative is to have a monorepo called nimbus.

But I share Stefan's concerns. However it is not a good time to change the repo name (audit, ...).

**Comment by sachayves** [comment]
@mratsim got it -- i agree those names are clearer, but perhaps not the right time. @stefantalpalaru, would changing the title of chapter 8 to 'Generating your keys with NBC` be good enough? or are there parts of the text that worry you too?

**Comment by stefantalpalaru** [comment]
See for yourself how many occurrences of "Nimbus" we have: https://gist.githubusercontent.com/stefantalpalaru/233305df6c5f4075154ed52fdbd32eba/raw/42ea689950c821a9cb4ae67fe29aef46da5f8be1/Nimbus_occurrences.txt

And that's for a project repository called "nim-beacon-chain" and a main binary called "beacon_node".

We're to blame for this confusion on various levels, of course, but we need to somehow prevent perfectly understandable attempts at building and using the project called "nimbus" instead of this one.

We can start with a Google search for "nimbus ethereum" and see what results pop up first, to replicate the user experience.

**Comment by sachayves** [comment]
@stefantalpalaru understood. I think I would lean towards addressing this in a separate PR — once we agree on what the appropriate terminology should be. 

As it stands, I think most people who are directed to this book associate nimbus with the eth2 client, so it’s not a disaster, but we should explore this more deeply.

Any other suggestions or objections? cc @arnetheduck, @mratsim, @zahary

File: docs/the_nimbus_book/src/medalla.md [hunk 1]
```diff
@@ -143,3 +142,53 @@ You may need to do this if you are running another client.
...
+
+### Upgrading
+
+When you restart the beacon node, the software will resume from where it left off, using your previous deposits.
+
+```
+cd nim-beacon-chain
+git pull
+make update # Update dependencies
+make altona # Restart using same keys as last run
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/medalla.md:154]:
```suggestion
make medalla # Restart using same keys as last run
```

**Review by arnetheduck — APPROVED** [review]

**Comment by zah** [comment]
I personally always try to use the name "Nimbus" when I discuss the project with users and other third-party developers. That's the "brand" that we want to develop. On a technical level, I think we can get there my introducing a `nimbus` binary eventually that will be capable of launching both Eth1 and Eth2 clients. I've written a bit more about it here:
https://github.com/status-im/nim-beacon-chain/issues/1539

**Pull request merged**